test/e2e/conditions.go

```diff
@@ -6,10 +6,53 @@ import (
 	corev1 "k8s.io/api/core/v1"
 )
 
-func isProvisioned() toolchainv1alpha1.Condition {
+func provisioned() toolchainv1alpha1.Condition {
```

Comment:
what's wrong with `isProvisioned`?

Reply:
1. `isProvisioned()` doesn't return a bool.
2. `provisioned()` looks better in the constructions where it's used. For example: `wait.UntilMasterUserRecordHasConditions(provisioned())`

It doesn't have to be a function btw. Could be a const. But IMO it's minor.

Reply:
ok, I see what you mean. I still believe that `isProvisioned` looks better, but it's cosmetic.